conda/plugins/manager.py

```diff
@@ -303,6 +311,10 @@ def get_subcommands(self) -> dict[str, CondaSubcommand]:
     def get_virtual_packages(self) -> tuple[CondaVirtualPackage, ...]:
         return tuple(self.get_hook_results("virtual_packages"))
 
+    def invoke_health_checks(self, prefix: str, verbose: bool) -> None:
+        for hook in self.get_hook_results("health_checks"):
+            hook.action(prefix, verbose)
```

Comment:
It'd be good to catch exceptions here, so health checks aren't blocking each other and we continue on with the list if one of them fails for some reason, like so:
```suggestion
        for hook in self.get_hook_results("health_checks"):
            try:
                hook.action(prefix, verbose)
            except Exception as err:
                log.warning(
                    f"Error running health check: {hook.name} ({err})"
                )
                continue

```

Reply:
Good idea.